src/styles/fonts.scss

```diff
@@ -0,0 +1,84 @@
+/* latin */
+@font-face {
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: local('Montserrat Regular'), local('Montserrat-Regular'), url('~@/fonts/Montserrat/Montserrat-Regular.woff2') format('woff2');
```

Comment:
I get the warning `Failed to decode downloaded font` in Chrome and `downloadable font: rejected by sanitizer` in Firefox for every font. It seems as though it could be related to a corrupt font file, or not having an alternate version of the font files. I'm not sure why I would be having this issue and not others, so maybe another reviewer can take a look.

Reply:
I believe it's fixed now.

Reply:
[LARGE-QUOTES]fixed[/LARGE-QUOTES]